Kulumbegov A.I.

```diff
@@ -0,0 +1 @@
+Student of eltech. 17 y.o. tryed java
```

Comment:
This file should be created in your own directory, not in root
See other for example

Reply:
Still not fixed